src/webhooks/handleInvoiceCompleted.ts

```diff
@@ -103,9 +103,13 @@ export default async function handleInvoiceCompleted(
   try {
     const userActiveSubscription = await paymentService.getActiveSubscriptions(customer.id);
     const hasActiveSubscription = userActiveSubscription.length > 0;
```

Comment:
Due to the changes, this check is no longer needed, as the case `hasIndividualActiveSubscription` is more specific than the `hasActiveSubscription` one

Reply:
Done